### PR TITLE
Disable bdsmodear when using widelane with BDS

### PR DESCRIPTION
**Pull request by dgburr**
This solves the performance regression identified in #72 when B2a signals are used: `bdsmodear` needs to be disabled when using `wide-lane` ambiguity resolution with BDS signals

### CODE REVIEW

File: app/consapp/convbin/convbin.c [hunk 1]
```diff
@@ -58,70 +58,70 @@
...
-    "",
-    " If not any output file is specified, default output files <file>.obs,",
-    " <file>.nav, <file>.gnav, <file>.hnav, <file>.qnav, <file>.lnav and",
-    " <file>.sbs are used. Empty output files are deleted after processing.",
-    "",
-    " If log format type is not specified, format type is recognized by the input",
-    " file extension as following:",
-    "     *.sbp        Swift Navigation SBP",
-    "     *.json       Swift Navigation SBP-JSON",
+"",
```
Comment (tao-swift) [inline app/consapp/convbin/convbin.c:61]: nit, what format should we follow in here. It seems `rtkrcv` uses indented format, but `rnx2rtkp` doesn't.

File: src/rinex.c [hunk 2]
```diff
@@ -1014,8 +1014,8 @@ static int readrnxobsb(FILE *fp, const char *opt, double ver, int *tsys,
             data[n].time=time;
             data[n].sat=(uint8_t)sats[i-1];
             
-            /* decode obs data */
-            if (decode_obsdata(fp,buff,ver,mask,index,data+n)&&n<MAXOBS) {
+            /* decode RINEX observation data */
+            if (decode_obsdata(fp,buff,ver,mask,index,data+n)) {
```
Comment (tao-swift) [inline src/rinex.c:1018]: any reason why we are dropping the second condition?

File: app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf [hunk 3]
```diff
@@ -70,7 +70,7 @@ pos2-armode        =wide-lane  # (0:off,1:continuous,2:instantaneous,3:fix-and-h
 # SWIFT: only enable for static positioning mode
 pos2-wlarmode      =off        # (0:off,1:on)
 pos2-gloarmode     =off        # (0:off,1:on)
-pos2-bdsarmode     =on         # (0:off,1:on)
+pos2-bdsarmode     =off        # (0:off,1:on)
```
Comment (tao-swift) [inline app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf:73]: nit, I guess here it depends on the customers' need. With the change, it defaults to no BDS AR.

File: app/consapp/convbin/convbin.c (hunk 1 above)
Comment (dgburr) [reply app/consapp/convbin/convbin.c:61]: This is the indentation used upstream

File: src/rinex.c (hunk 2 above)
Comment (dgburr) [reply src/rinex.c:1018]: It does not exist upstream

File: app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf (hunk 3 above)
Comment (dgburr) [reply app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf:73]: Correct.  The whole point of this PR is to disable BDS AR when widelane is enabled.

Comment (dgburr) [reply app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf:73]: Alternatively we can enable `pos2-bdsarmode` by default but remove BDS from `pos1-navsys`.  Then in the App Note I can add a comment saying to disable `pos2-bdsarmode` if BDS is added to `pos1-navsys`.

Comment (tao-swift) [reply app/consapp/rtkrcv/conf/rtk_skylark_l1l5.conf:73]: If we don't plan on using BDS, i think we can remove BDS from pos1-navsys. With the current configuration as off, it means BDS will still be used in the float solution, as the second case in the following.
1 RTK with no BDS
2 RTK with BDS in float mode
3 RTK with BDS in fixed mode

**Review by tao-swift — APPROVED** [review]
lgtm!

**Pull request merged**